### ISSUE TITLE
Archaic accent frequently results in an ungrammatical mess

### ISSUE THREAD

**Issue by arimah** [space-wizards/space-station-14#19208]
The archaic accent contains a considerable number of replacements, almost all of which are wrong in some way. Old Modern English and Early Modern English had a more complex grammar than mere "dumb" word substitutions can encompass. I understand that the accent system is not designed to be perfect, and some leeway must be given to that. Nevertheless, here are some of the more egregious substitutions and their potential "archaic" gibberish:

"get" → "secure": _get_ as a verb has dozens of meanings, very few of which can be translated as "secure": https://en.wiktionary.org/wiki/get#Verb

> I get tired of greys → I secure tired of greys
> I don't get it → I don't secure it
> When did you get here? → When did thou secure here?

Everything related to you/your/yours → thou/thy/thine is wrong. "Thou" is strictly singular and nominative, whereas (modern) "you" is singular, plural, nominative and oblique all in one. Also, "thy" is used before consonants, "thine" before vowels.

> all of you → all of thou _(correct: all of you)_
> you are all good → thou art all well _(correct: ye are all good — 'well' as an adjective means 'of good health', and is otherwise an adverb)_
> I don't know you → I don't know thou _(correct, singular: I don't know thee; plural: I don't know you)_
> get your emag from your backpack → secure thy emag from thy backpack _(more correct, assuming singular: get/acquire/obtain/produce/bring forth thine emag from out thy rucksack)_

In some cases, actual archaic words are replaced with more modern variants: "ilk" → "consort" is such an example. Another weird one is "why" → "for what reason", whereas "wherefore" would have made more sense as a near-perfect drop-in replacement.

Some replacements are just plain semantically wrong, or ignore actual important verb/noun distinctions. A smattering of examples:

> where → whither _('whither' means 'where to')_
> happened → befall'n _(what befell 'befell'?)_
> need → require _(what of 'requirement'?)_
> by chance → perchance _('perchance' means 'maybe', which is only one meaning of 'by chance')_
> sure → alright _(I'm ~~sure~~ alright that won't cause confusion)_
> full → fraught _(these words overlap in only one sense, and are otherwise quite distinct)_
> can → may _('can' also means 'to be able to', which 'may' does not)_
> can i → am i allowed to _(see above)_
> song → catch _(a catch is but part of a refrain; the actual archaic word for 'song' is 'song')_
> please → i request of thee _(seriously?)_
> was → have been _(no.)_

"can i please get it back" stands a chance of being mangled into the utterly incomprehensible "am i allowed to i request of thee secure it back".

To summarise: I feel like the archaic accent could stand to be improved somewhat. A perfect archaic English is impossible to emulate, but too many of the substitutions are overly broad or flat out wrong.

**Comment by arimah** [comment]
I want to underscore that despite the fairly complainy tone of my remarks above, I have the utmost respect for you and your contribution. I intend everything to be read in a spirit of collaboration and desire to improve upon what has rapidly transformed into one of my favourite games :)

I'm glad my words did not upset; getting critiqued by some random stranger can be weird.

**Comment by arimah** [comment]
The archaic accent has been removed. This issue is no longer relevant.
